src/routes/views/components/charts/trendChart/trendChart.tsx

```diff
@@ -310,7 +311,7 @@ class TrendChartBase extends React.Component<TrendChartProps, State> {
     return result;
   }
 
-  private getLegend = () => {
+  private getLegend = (chartName = 'chart') => {
     const { legendItemsPerRow } = this.props;
```

Comment:
This could be simplified by adding `chartName` to the destructured props on line 315.
```
    const { chartName, legendItemsPerRow } = this.props;
```